src/Datatype.cpp

```diff
@@ -303,4 +303,90 @@ namespace openPMD
         os << dt;
         return buf.str();
     }
+
+    std::vector<Datatype> openPMD_Datatypes{
+           Datatype::CHAR ,
```

Comment:
do the arguments here have to correspond to the exact order and number of arguments as elsewhere? Can we bring them closer together or at least cross-link them via a doxygen comment above each of them?

Reply:
The declaration is in [Datatype.hpp](https://github.com/franzpoeschel/openPMD-api/blob/909fa5efcd6a692535aa1cee4f905b6d439b7b24/include/openPMD/Datatype.hpp#L700). I think, it's better to move it below the `Datatype` enum, will do that.